### PR TITLE
Fix `export_asce` taking into account the corner case using the `default` site class

### PR DESCRIPTION
**Pull request by ptormene**
Weekly tests were broken. I am running them here:
https://github.com/gem/oq-engine/actions/runs/30365478084

### CODE REVIEW

File: openquake/calculators/export/hazard.py [hunk 1]
```diff
@@ -806,7 +806,14 @@ def export_mce(ekey, dstore):
 def export_asce(ekey, dstore):
     sitecol = dstore['sitecol']
     for s, site in enumerate(sitecol):
```
Comment (micheles) [inline openquake/calculators/export/hazard.py]: I don't like this. Something like
```python
if len(sitecol) == 3 and len(dstore[ekey[0]) == 1:
   # default site class case
   sids = [0]
else:
   sids = sitecol.sids
for s in sids:
      ...
```
would be better

Comment (ptormene) [reply openquake/calculators/export/hazard.py]: Done, thanks

**Review by micheles — APPROVED** [review]
